Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/JPath/gen.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+java \
```

Comment:
Since we do not use antlr as a nuget or npm package the automated dependency tracker is not aware.

Could you manually register usage of antlr in component covernance before checking in?

Reply:
We still use the antlr runtime from Nuget.   I'll double check if that's the same jar file as the one used for generating parser/lexer and if it is not I'll register this one manually